src/components/OneToDo/OneToDo.tsx

```diff
@@ -2,19 +2,30 @@ import { Checkbox, IconButton, TextField, Typography } from "@mui/material";
 import DeleteIcon from "@mui/icons-material/Delete";
 import EditIcon from "@mui/icons-material/Edit";
 import SaveIcon from "@mui/icons-material/Save";
-import React, { useRef } from "react";
+import React, { KeyboardEventHandler, useRef } from "react";
 import { useState } from "react";
 import "./OneToDo.css";
+import { ToDo } from "../../types/types";
 
-function OneToDo({ index, toDoList, editToDo, deleteToDo }) {
+function OneToDo({
+  index,
+  toDoList,
+  editToDo,
+  deleteToDo,
+}: {
+  index: number;
+  toDoList: ToDo[];
+  editToDo: any;
+  deleteToDo: any;
```

Comment:
instead of defining types inside the funciton arguments you can create a propType object and define types there. like :

Interface OneTodoPropsType{
  index: number;
  toDoList: ToDo[];
  editToDo: any;
  deleteToDo: any;
  }
  
  function OneToDo({
  index,
  toDoList,
  editToDo,
  deleteToDo,
}: OneTodoPropsType)

Reply:
why are we using any type for editTodo and deleteTodo?
Never use any type. Make custom types for these

Reply:
Update the same thing at other places too